.github/workflows/rust.yml

```diff
@@ -163,14 +163,15 @@ jobs:
     - name: Check no-std
       run: |
         cd ensure-no_std
-        cargo build --release
+        cargo build --no-default-features
+        cargo build
 
   tests:
     needs: build-programs
     strategy:
       fail-fast: false
       matrix:
-        special-features: ["", "lambdaworks-felt"]
+        special_features: ["", "lambdaworks-felt"]
```

Comment:
is ok that you also changed the underscore?

Reply:
Yes. This was, in fact, a typo in the original PR. The typo was causing the CI to run tests without the `lambdaworks-felt` feature.